app/validators/form_dob_validator.rb

```diff
@@ -0,0 +1,44 @@
+module FormDobValidator
+  extend ActiveSupport::Concern
+  include ActionView::Helpers::TranslationHelper
+
+  included do
+    include ActiveModel::Validations::Callbacks
+    private attr_accessor :dob_original
+
+    # using rails 7 built-in Comparison validator
+    validates_comparison_of :dob, less_than_or_equal_to: ->(_rec) {
```

Comment:
I feel like this comment is not helpful? like describing which version is useful sometimes when config values change but this is a method that's easily searchable

```suggestion
    validates_comparison_of :dob, less_than_or_equal_to: ->(_rec) {
```

Reply:
@zachmargolis, as previous comment mentioned, this was removed during refactor process.